### ISSUE TITLE
Designer crashes System.InvalidOperationException: DPI mode value - DpiUnaware

### ISSUE THREAD

**Issue by pascallauer** [dotnet/winforms#9860]
### Environment

Microsoft Visual Studio Professional 2022 (64-Bit) - Preview
Version 17.8.0 Preview 1.0

### .NET version

net7.0-windows7.0

### Issue description

I am using VS2022 for developing a Windows Forms app. Since i have a 4k screen i have a scaling of 150%. To keep my apps working on different DPI Scalings i have to restart visual studio everytime or set a registry key to disable the automatic DPI adjustment.

To do this, i use a registry setting: 

[Computer\HKEY_CURRENT_USER\Software\Microsoft\Windows NT\CurrentVersion\AppCompatFlags\Layers]
"C:\\Program Files\\Microsoft Visual Studio\\2022\\Preview\\Common7\\IDE\\devenv.exe"="DPIUNAWARE" 

### Steps to reproduce

The Issue was already well descriped in the following issue:
https://github.com/dotnet/winforms/issues/7609

If i remove the registry key and start vs with the /noscale parameter everything seems to work fine. 
But that's not working for the "recent projects" in the taskbar.

### Diagnostics

```text
[13:54:05.373798] warn: DPI mode SystemAware specified for the project is different from the DPI mode specified for the designer. Winforms designer will have DPI mode DpiUnaware 
[13:54:08.069092] fail: [Cosmolink.Scripting.Lauchner]: Failed to launch design tools server process process
                        System.InvalidOperationException: Prozess-DPI-Moduswert konnte nicht auf eingestellt werden:DpiUnaware.
                           at Microsoft.WinForms.Utilities.Desktop.DpiHelper.SetDesignerProcessDpiAwareness(DpiAwarenessContext dpiContext)
                           at Microsoft.DotNet.DesignTools.Server.Program.<>c__DisplayClass2_0.<<Main>b__1>d.MoveNext()

                        Informationen zur Problembehandlung für den Designer finden Sie im Leitfaden unter https://aka.ms/winforms/designer/troubleshooting.
[13:56:07.733097] fail: [Cosmolink.Scripting.Lauchner]: Timed out while connecting to named pipe: DesignToolsServer.e550f24b-2700-43d0-b492-f84c283e2342

                        Informationen zur Problembehandlung für den Designer finden Sie im Leitfaden unter https://aka.ms/winforms/designer/troubleshooting.
[13:56:07.738130] fail: Microsoft.DotNet.DesignTools.Client.ServerException: Timeout beim Herstellen einer Verbindung mit Named Pipe.
                           bei Microsoft.DotNet.DesignTools.Client.Host.ServerProcess.<ConnectToStreamAsync>d__36.MoveNext()
                        --- Ende der Stapelüberwachung vom vorhergehenden Ort, an dem die Ausnahme ausgelöst wurde ---
                           bei System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
                           bei System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
                           bei Microsoft.DotNet.DesignTools.Client.Host.ServerProcess.<LaunchAsync>d__28.MoveNext()
                        --- Ende der Stapelüberwachung vom vorhergehenden Ort, an dem die Ausnahme ausgelöst wurde ---
                           bei System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
                           bei System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
                           bei Microsoft.DotNet.DesignTools.Client.Host.ServerHostFactory.<CreateHostAsync>d__8.MoveNext()
                        --- Ende der Stapelüberwachung vom vorhergehenden Ort, an dem die Ausnahme ausgelöst wurde ---
                           bei System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
                           bei System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
                           bei Microsoft.DotNet.DesignTools.Client.DesignToolsClientLoader.<CreateClientAsync>d__29.MoveNext()

                        Informationen zur Problembehandlung für den Designer finden Sie im Leitfaden unter https://aka.ms/winforms/designer/troubleshooting.
[13:56:08.023776] warn: Cannot update selection in server process as Session is not connected.
```

**Comment by Olina-Zhang** [comment]
A dup of GH issue: **https://github.com/dotnet/winforms/issues/9714**, which was fixed in next 17.8 Preview release.

**Comment by JeremyKuhne** [comment]
Duplicate of #9714
